test/e2e/create_test.go

```diff
@@ -104,7 +104,7 @@ func (tc *testContext) createWindowsMachineSet(replicas int32, windowsLabel bool
 // immediately, else we will wait for the duration given by nodeCreationTime variable which is 20 minutes increasing
 // the overall wait time in test suite
 func (tc *testContext) waitForWindowsNodes(nodeCount int32, waitForAnnotations, expectError, checkVersion bool) error {
-	var nodes *v1.NodeList
+	var nodes *core.NodeList
```

Comment:
This should probably be a separate commit as its an unrelated change

Reply:
I suggest leaving this out.